docker/pytorch-aarch64/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ where `YY` is the year, and `MM` the month of the increment.
 https://github.com/ARM-software/Tool-Solutions/tree/tensorflow-pytorch-aarch64--r23.05/docker/pytorch-aarch64
 
 ### Added
+- Adds tensor dilation parameter configuration for acl depthwise convolution
```

Comment:
This CHANGELOG change is in the wrong place - these changes aren't part of the 23.05 increment. They should go in the [Unreleased] section initially, and will then be included in the changes for the next release.